### PR TITLE
Feat/show basic user page

### PR DESCRIPTION
**Pull request by ImMPrada**
This PR:
- adds a scope to the Event model, to distinguish expired events and active events
- events#index lists events separate by active and expired events
   image
- adds link to the user profile
   image
- user's profile shows user grouped by active and expired events
   image

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -10,6 +10,8 @@ def show
 
   def index
     @events = Event.all.includes(:creator)
+    @future_events = @events.active
+    @past_events = @events.expired
```
Comment (mapra99) [inline app/controllers/events_controller.rb:14]: do you need to `includes(:creator)` on these too? That could be moved to another scope as well, something like
```ruby
scope :creator_included, -> { includes(:creator) }
```

```suggestion
    @future_events = @events.active.creator_included
    @past_events = @events.expired.creator_included
```

File: config/routes.rb [hunk 2]
```diff
@@ -8,4 +8,6 @@
   resources :events, only: %i[index show new create]
 
   resources :attendances, only: %i[create destroy], param: :event_id
+
+  resources :users, only: %i[show]
```
Comment (mapra99) [inline config/routes.rb]: this is fine but just so you know, sometimes I've seen this getting sorted in a different way. You can create a [single resource](https://guides.rubyonrails.org/routing.html#singular-resources) called `profile` 

```ruby
resource :profile, only: %i[show]
```

that way the request that is handled by `#show` is `GET /profile`. You don't need to pass an ID on the URL

**Review by mapra99 — APPROVED** [review]

File: config/routes.rb (hunk 2 above)
Comment (ImMPrada) [reply config/routes.rb]: nice! done!

File: app/controllers/events_controller.rb (hunk 1 above)
Comment (ImMPrada) [reply app/controllers/events_controller.rb:14]: `@event` already includes :creator

**Pull request merged**